test/unit/action/wallet.spec.js

```diff
@@ -354,13 +354,16 @@ describe('Action Wallet Unit Tests', () => {
       expect(nav.goHome, 'was called once');
     });
 
-    it('should display error notification on failure', async () => {
+    it('should display error notification on failure and clear password', async () => {
+      wallet.setPassword('not-empty');
+      expect(store.wallet.password, 'not to be', '');
       grpc.sendUnlockerCommand
         .withArgs('UnlockWallet')
         .rejects(new Error('Boom!'));
       await wallet.unlockWallet({ walletPassword: 'baz' });
       expect(notification.display, 'was called once');
       expect(nav.goWait, 'was not called');
+      expect(store.wallet.password, 'to be', '');
```

Comment:
@tanx I've updated the PR according the to feedback, but I want to point out that the logic for `setPassword` is already tested in a separate unit test.

https://github.com/lightninglabs/lightning-app/blob/master/test/unit/action/wallet.spec.js#L82-L87